### PR TITLE
Modified rkt pods and images garbage collection

### PR DESCRIPTION
**Pull request by zed-rishabh**
1. Moved rkt garbage collection from domainmgr to verifier
2. Increased grace time from 5 minutes to 24 hours
3. Increased minimum grace time from 30 seconds to 12 hours

Signed-off-by: zed-rishabh <rgupta@zededa.com>

### CODE REVIEW

**Comment by zed-rishabh** [comment]
@kalyan-nidumolu As discussed moved rkt garbage collection from domainmgr to verifier and increased grace time from 5 minutes to 24 hours. Please take a look. Thanks!

File: pkg/pillar/types/global.go [hunk 1]
```diff
@@ -332,7 +332,7 @@ var GlobalConfigMinimums = GlobalConfig{
 	DownloadRetryTime:       60,
 	DomainBootRetryTime:     10,
 	Dom0MinDiskUsagePercent: 20,
-	RktGCGracePeriod:        30,
+	RktGCGracePeriod:        43200,
```
Comment (eriknordmark) [inline pkg/pillar/types/global.go]: While 30 seconds might be too small as a minimum, having the min configrable value be 12 hours seems excessive. Wouldn't a min of 5 or 10 minutes make more sense?

Comment (zed-rishabh) [reply pkg/pillar/types/global.go]: @eriknordmark Yes, we can set min value to 10 min. I will update it. Thanks.

File: docs/CONFIG-PROPERTIES.md [hunk 2]
```diff
@@ -12,7 +12,7 @@
 | timer.use.config.checkpoint | integer in seconds | 600 | use checkpointed config if no cloud connectivity |
 | timer.gc.download | integer in seconds |  600 | garbage collect unused downloaded objects |
 | timer.gc.vdisk | integer in seconds | 1 hour | garbage collect unused instance virtual disk |
-| timer.gc.rkt.graceperiod | integer in seconds | 300 | grace period to be used in rkt gc command |
+| timer.gc.rkt.graceperiod | integer in seconds | 86400 | grace period to be used in rkt gc command |
```
Comment (kalyan-nidumolu) [inline docs/CONFIG-PROPERTIES.md]: As discussed, make the default value 1 hour

File: pkg/pillar/types/global.go [hunk 3]
```diff
@@ -203,12 +203,12 @@ var GlobalConfigDefaults = GlobalConfig{
...
-	VdiskGCTime:         3600, // 1 hour
-	DownloadRetryTime:   600,  // 10 minutes
-	DomainBootRetryTime: 600,  // 10 minutes
-	RktGCGracePeriod:    300,  // 5 minutes
+	StaleConfigTime:     600,   // Use stale config for up to 10 minutes
+	DownloadGCTime:      600,   // 10 minutes
+	VdiskGCTime:         3600,  // 1 hour
+	DownloadRetryTime:   600,   // 10 minutes
+	DomainBootRetryTime: 600,   // 10 minutes
+	RktGCGracePeriod:    86400, // 24 hours
```
Comment (kalyan-nidumolu) [inline pkg/pillar/types/global.go]: Change this to 1 hour..

File: docs/CONFIG-PROPERTIES.md (hunk 2 above)
Comment (zed-rishabh) [reply docs/CONFIG-PROPERTIES.md]: Resolved.

File: pkg/pillar/types/global.go (hunk 3 above)
Comment (zed-rishabh) [reply pkg/pillar/types/global.go]: Resolved.

**Pull request merged**